### PR TITLE
add qstat -q message fix for issue #11

### PR DESCRIPTION
**Pull request by hajgato**

### CODE REVIEW

**Comment by boegel** [comment]
@hajgato While you're at it, can we also filter out the `special` queue from the `qstat -q` output?

```
$ qstat -q
Queue            Memory CPU Time Walltime Node  Run Que Lm State
---------------- ------ -------- -------- ----  --- --- -- -----
victini            --      --    72:00:00  --    0   0 --  E R
victini_special    --      --    144:00:00  --    0   0 --  E R
                                               ----- -----
                                                   0     0
```

(and fix the alignment of the columns ;) )

**Comment by hajgato** [comment]
fixes issue #28 as well

File: files/qstat.pl [hunk 1]
```diff
@@ -653,15 +658,15 @@ sub print_part_limits
 
     printf("%-16.16s   --      --    ", $part->{name});
     if($part->{max_time} != INFINITE) {
-        print(hhmmss($part->{max_time}*60));
+        printf("%8.8s", hhmmss($part->{max_time}*60)));
```
Comment (boegel) [inline files/qstat.pl]: @hajgato You brokes it! You have an extra `)` here...

```
$ ./qstat.pl
syntax error at ./qstat.pl line 661, near "))"
Execution of ./qstat.pl aborted due to compilation errors.
```

File: files/qstat.pl [hunk 2]
```diff
@@ -129,10 +129,12 @@ sub main
             }
 
             if ($full) { # Full
-                print_part_full($part);
+                print_part_full($part) if $part->{name} !~ m/_special$/;
             } else { # Brief
-                print_part_brief($part, $line);
-                $line++;
+                if ($part->{name} !~ m/_special$/) {
+                    print_part_brief($part, $line) if $part->{name} !~ m/_special$/;
```
Comment (boegel) [inline files/qstat.pl]: the `if ...` part at the end is useless here, already done above?

**Pull request merged**